### PR TITLE
LPS-198962

### PR DESCRIPTION
**Pull request by icgsbr**
# Jira Link
https://liferay.atlassian.net/browse/LPS-198962

# Old Pull Request
https://github.com/liferay-forms/liferay-portal/pull/2385

### CODE REVIEW

**Comment by icgsbr** [comment]
ci:test:stable

**Comment by icgsbr** [comment]
ci:test:relevant

**Comment by icgsbr** [comment]
ci:test:forms

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/utils/evaluation.es.js [hunk 1]
```diff
@@ -181,7 +181,7 @@ const doEvaluate = debounce((fieldName, evaluatorContext, callback) => {
 			callback(null, mergedPages);
 		})
 		.catch((error) => callback(error));
-}, 600);
+}, 200);
```
Comment (cgmonte) [inline modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/utils/evaluation.es.js:184]: we are playing timing games here. in the previous solution added a time out, but the bug persisted if the user is faster enough, now we are reducing delay. we should seek a different solution. lets work on this together.

Comment (icgsbr) [reply modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/utils/evaluation.es.js:184]: Perfect! Thanks for the heads-up! I'll reach you on Slack so we can schedule a meeting. In the meantime, I'll be working on finding a different approach.

**Review by cgmonte — APPROVED** [review]
Well, after spending another 4 hours on this with @icgsbr, we could not come up with a better solution, and this one works, so I will approve this and move on.

**Comment by cgmonte** [comment]
ci:forward

**Pull request closed without merging**